### ISSUE TITLE
Fancier Logo

### ISSUE THREAD

**Issue by call-me-matt** [julien-nc/cospend-nc#290]
what about a fancier logo?
As I am not a designer, here some inspiration from DALL-E...

![image](https://github.com/user-attachments/assets/6dc893a5-919e-4e6d-8ffa-789929db22c9)

![image](https://github.com/user-attachments/assets/9f053f22-19c0-4133-a923-3f13db45c9a4)

![image](https://github.com/user-attachments/assets/16fb9b55-d898-4233-b843-8bdda129cdc7)

**Comment by MrRinkana** [comment]
These are not good logos:
-They are too complex, remember the logo will be rather small. Resize them so that they are (on your screen) as small as the logos visible in the menu bar in nc and you'll understand what I mean.
- There is too much colour, Nextcloud app logos are monotone so that the theme can adapt them such that they work with the theme colours and the background (they are black in light themes, white in dark themes)
- Logos are in a way a projects identity, generated content might not be a good fit for such purposes.
- Logos are a trademark, while generated content must be public domain and hence not compatible

**Comment by call-me-matt** [comment]
I agree to all of your points, but stay with my opinion that the logo could be more unique. A money bag can stand for so many things and is okay as an icon, but not a logo. Of course, the dall-e pictures are no good logos neither and just a source of inspiration. I wanted to stay constructive (without the capabilities to be productive here).
